skyVegasTests/utilities/makeSpinHelper.ts

```diff
@@ -0,0 +1,12 @@
+import { type Page } from "@playwright/test";
+import { validateConsoleMessages } from "./validateConsoleMessagesHelper";
+
+export const makeSpin = async (page:Page, expectedMessage:string, consoleMessages:string[]) => {
```

Comment:
Similarly to the rename function comment below, maybe this could be called makeSpinWithEnterKey or something along those lines to make it more specific - if possible we're going to be attempting a general makeSpin later which would look different to this